runtime/polkadot/src/lib.rs

```diff
@@ -1592,4 +1594,18 @@ mod test_fees {
 		println!("can support {} nominators to yield a weight of {}", active, weight_with(active));
 		assert!(active > target_voters, "we need to reevaluate the weight of the election system");
 	}
+
+	#[test]
+	fn vesting_schedules_encoded_use_correct_space() {
+		use pallet_vesting::vesting_info::VestingInfo;
+		use frame_support::BoundedVec;
+		use parity_scale_codec::MaxEncodedLen;
+
+		let sched_len = VestingInfo::<Balance, BlockNumber>::max_encoded_len();
+		let vec_len = BoundedVec::<VestingInfo<Balance, BlockNumber>, MaxVestingSchedules>::max_encoded_len();
+
+		println!("Vesting schedules max encoded len: {} ({}).", vec_len, sched_len);
+		assert_eq!(sched_len, 36);
+		assert_eq!(vec_len, 1009);
+	}
```

Comment:
Not sure if I should leave this test in here

Reply:
This seems like it's served its purpose and isn't really testing anything useful here anymore, so I'd recommend removing it.